node/clients/catalog.ts

```diff
@@ -214,7 +214,7 @@ export class Catalog extends AppClient {
       const segmentData = (this.context as CustomIOContext).segment
       salesChannel = (segmentData && segmentData.channel.toString()) || ''
     }
-    let url = `/pub/products/search/${sanitizedQuery}?`
+    let url = `/pub/products/search/${sanitizedQuery}`
```

Comment:
I think this is wrong

Reply:
If this breaks you, you should then check if `?` is the last character of `sanitizedQuery` or not before removing it